scripts/envhook.py

```diff
@@ -41,6 +41,18 @@ def main(argv):
         else:
             raise
     if options.action == 'install':
+        try:
+            import sitecustomize
+        except ImportError:
+            pass
+        else:
+            try:
+                from sitecustomize import setenv
```

Comment:
I'd prefer a stronger signal

Reply:
Like comparing `__file__`, if necessary with os.realpath.

Reply:
Good point! I forgot that one could do this.